app/models/payment.rb

```diff
@@ -100,4 +105,12 @@ def successfully_completed
     notify_observers(self)
   end
 
+
+  def note_payor_deleted(payor_type = NOTES_DEFAULT_PAYOR,
+                         payor_email = NOTES_UNKNOWN_EMAIL,
+                         deleted_time = Time.now.utc)
```

Comment:
I suspect you want to store time in UTC in order to facilitate comparison against times in HIPS(?).

I think we may want to store the time in the _users's_ TZ.  That would help us (and users) better understand when things have happened and address any questions that might arise (this is speculative, of course, but I don't see any upside to using UTC time).  We can always convert to UTC time for time comparisons with HIPS.

Reply:
I was thinking about comparing it with the HIPS payments and that keeping it in UTC would make any searching and comparison easier.  But it's a minimal gain, to be sure.

I'll switch it to use `Time.zone.now` per our wiki page (https://github.com/AgileVentures/shf-project/wiki/Dates-and-Times-and-Timezones-(oh-my!) ).

We don't really have a way to determine a user's time zone ATM -- except for the fact that we happen to know where the administrator lives and what time zone she's in. ;-)